store/migration/10001__init_schema.sql

```diff
@@ -829,7 +829,7 @@ CREATE TABLE task (
     pipeline_id INTEGER NOT NULL REFERENCES pipeline (id),
     stage_id INTEGER NOT NULL REFERENCES stage (id),
     instance_id INTEGER NOT NULL REFERENCES instance (id),
-    -- Could be empty for tasks like creating database
+    -- could be empty when the task status is not 'DONE'
```

Comment:
// Could be empty for creating database task when the task isn't yet completed successfully.